packages/calcite-tailwind-preset/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@esri/calcite-tailwind-preset",
+  "version": "1.0.0",
```

Comment:
Can you set the version to `0.0.0`? Otherwise I think release-please/lerna will version from `1.0.0` and end up as `1.1.0`.

Reply:
Per our convo, changed to `0.1.0`.